docs/content/docs/_index.md

```diff
@@ -1045,6 +1045,15 @@ Example: `{{ value | urlencode }}`
 
 If value is `/foo?a=b&c=d`, the output will be `/foo%3Fa%3Db%26c%3Dd`. `/` is not escaped.
 
+#### urlencode_strict
+Only available if the `builtins` feature is enabled.
+
+Similar to `urlencode` filter, percent-encodes all non-alphanumeric characters in a string.
```

Comment:
it would be useful to mention the difference explicitely here.

Reply:
Sure updated the docs. Let me know if you would like to reword/add something more here.